MacrobenchmarkSample/macrobenchmark/src/main/java/com/example/macrobenchmark/TrivialBaselineProfileBenchmark.kt

```diff
@@ -24,7 +24,9 @@ import org.junit.Test
 
 @OptIn(ExperimentalBaselineProfilesApi::class)
 // Remove `@Ignore`` to actually run the test using an emulator without Google Play Services
+// This is because we need to run the test on a user-debug build.
```

Comment:
Rephrase this, to focus on the rootability part:

// Remove `@Ignore` to actually run the test using a rooted device/emulator.
// Generally emulator images without Google Play Services are userdebug builds, which support `adb root`